tk_toolchain/cmd_line_tools/tk_docs_preview/__init__.py

```diff
@@ -181,7 +181,7 @@ def main(arguments=None):
                 import PySide  # noqa
             except ImportError:
                 try:
-                    import PySide2
+                    import PySide2  # noqa
```

Comment:
Do we conventionally give a reason for these, or skip doing so if it's obvious?  i.e.

```suggestion
                    import PySide2  # noqa testing importability, ignore unused import
```